htdocs/core/class/smtps.class.php

```diff
@@ -514,8 +514,9 @@ private function _server_authenticate()
                     $_retVal = $this->socket_send_str(base64_encode($this->_smtpsPW), '235');
                     break;
             }
-            if(!$_retVal)
+            if (! $_retVal) {
```

Comment:
OK :) a lot of other test are done with `{` `}`  i use same behaviour.

Please : you can just review and ask me to update next time :) :heart: